apps/src/templates/lessonOverview/activities/ProgressionDetails.jsx

```diff
@@ -25,6 +26,17 @@ export default class ProgressionDetails extends Component {
 
   handleBubbleClick = level => {
     this.setState({previewingLevel: level});
+    firehoseClient.putRecord(
+      {
+        study: 'lesson-plan',
+        study_group: 'teacher-lesson-plan',
+        event: 'click-level',
+        data_json: JSON.stringify({
+          levelId: level.id
```

Comment:
This will be the script level id, unlike above which will be the actual level id. Not sure which one we want but probably worth it for them to be the same?

Reply:
(Also apologies for the bad variable name here...)

Reply:
Alright I'll make the one above script level too